docs/cli-client/bank/send.md

```diff
@@ -2,16 +2,14 @@
 
 ## Description
 
-Sending tokens to another address. 
+Sending tokens to another address， this command includes "create/sign/broadcast" transaction.
```

Comment:
, this command includes `create/sign/broadcast` transaction.

Reply:
fixed